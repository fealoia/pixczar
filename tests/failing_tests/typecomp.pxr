Void main() {
	Int x = 3;
	String y = "three";
	print(x > y);
}
