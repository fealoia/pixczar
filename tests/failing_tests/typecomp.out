Fatal error: exception Failure("illegal binary operator Int > String in x > y")
